src/main/java/org/folio/rest/impl/VoucherNumberImpl.java

```diff
@@ -37,15 +37,39 @@ public void getVoucherStorageVoucherNumber(String lang, Map<String, String> okap
           if(reply.succeeded()) {
             String voucherNumber = reply.result().getList().get(0).toString();
             log.debug("Retrieved voucher number: {}", voucherNumber);
-            asyncResultHandler.handle(succeededFuture(respond200WithApplicationJson(new SequenceNumber().withSequenceNumber(voucherNumber))));
+            SequenceNumber sequenceNumber = new SequenceNumber().withSequenceNumber(voucherNumber);
+            asyncResultHandler.handle(succeededFuture(VoucherStorageVoucherNumber.GetVoucherStorageVoucherNumberResponse.respond200WithApplicationJson(sequenceNumber)));
           } else {
             throw new Exception(reply.cause());
           }
         } catch (Exception e) {
           log.error(e.getMessage(), e);
-          asyncResultHandler.handle(succeededFuture(respond500WithTextPlain(messages.getMessage(lang, MessageConsts.InternalServerError))));
+          String msg = messages.getMessage(lang, MessageConsts.InternalServerError);
+          asyncResultHandler.handle(succeededFuture(VoucherStorageVoucherNumber.GetVoucherStorageVoucherNumberResponse.respond500WithTextPlain(msg)));
         }
       });
     });
   }
+
+  @Override
+  public void postVoucherStorageVoucherNumberStartByValue(String value, String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    vertxContext.runOnContext((Void v) -> {
+      if(NumberUtils.isDigits(value)) {
+        String tenantId = TenantTool.calculateTenantId(okapiHeaders.get(RestVerticle.OKAPI_HEADER_TENANT));
+        PostgresClient.getInstance(vertxContext.owner(), tenantId).execute(String.format(SET_START_SEQUENCE_VALUE_QUERY, value), reply -> {
+          if (reply.succeeded()) {
+            log.debug("(Re)set start value for voucher number sequence: {}", value);
+            asyncResultHandler.handle(succeededFuture(VoucherStorageVoucherNumber.PostVoucherStorageVoucherNumberStartByValueResponse.respond204()));
+          } else {
+            log.error(reply.cause().getMessage(), reply.cause());
+            String msg = messages.getMessage(lang, MessageConsts.InternalServerError);
+            asyncResultHandler.handle(succeededFuture(VoucherStorageVoucherNumber.PostVoucherStorageVoucherNumberStartByValueResponse.respond500WithTextPlain(msg)));
+          }
+        });
+      } else {
+        log.debug("Illegal value: {}", value);
+        asyncResultHandler.handle(succeededFuture(VoucherStorageVoucherNumber.PostVoucherStorageVoucherNumberStartByValueResponse.respond400WithTextPlain("Bad request - illegal value")));
```

Comment:
The response here is plain text which means that [integer raml type](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md#integer) could be used with `Validate` annotation